File: pgm/ABC_Visual_Basic/ABC_167_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        Dim T As String = Console.ReadLine()
        Console.WriteLine(If(S = T.Substring(0, T.Length - 1), "Yes", "No"))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_167_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        String S = Console.ReadLine();
        String T = Console.ReadLine();
        Console.WriteLine(S == T.Substring(0, T.Length - 1) ? "Yes" : "No");
        return;
    }
}